croud/__main__.py

```diff
@@ -1011,6 +1014,62 @@
                     },
                 },
             },
+            "secrets": {
+                "help": "Manage organization's secrets.",
+                "commands" : {
+                    "list": {
+                        "help": "Lists all the secrets that belong to an organization.",
+                        "extra_args": [
+                            Argument(
+                                "--org-id", type=str, required=True,
+                                help="The organization ID to use.",
+                            ),
+                        ],
+                        "resolver": org_secrets_list,
+                    },
+                    "create": {
+                        "help": "Creates a new secret for the given organization.",
+                        "extra_args": [
+                            Argument(
+                                "--org-id", type=str, required=True,
+                                help="The organization ID to use.",
+                            ),
+                            Argument(
+                                "--name", type=str, required=True,
+                                help="The name the Secret will be known as.",
```

Comment:
```suggestion
                                help="The name the secret will be known as.",
```
I would let it lowercase here, because it is in most of the help message you added.
Same a few lines bellow.